src/OrchardCore.Modules/OrchardCore.Users/Workflows/ViewModels/SelectUsersInRoleTaskViewModel.cs

```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace OrchardCore.Users.Workflows.ViewModels
+{
+    public class SelectUsersInRoleTaskViewModel
+    {
+        [Required]
+        public string PropertyName { get; set; }
```

Comment:
Rename to something other that PropertyName. I am not sure why this property is needed. Roles should be all you need.